tools/lint-pr-url.mjs

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+// Usage:
+// git diff upstream/master HEAD -G"pr-url:" -- "*.md" | \
+// ./tools/lint-pr-url.mjs <expected-pr-url>
+
+import process from 'node:process';
+import readline from 'node:readline';
```

Comment:
```suggestion
import readline from 'readline';
```
nit: since `process` is a part of the `global` object
question: why are we prepending `'node:'` to the imported module names?

Reply:
> nit: since `process` is a part of the `global` object

I try to not rely on Node.js-only globals when writing ES modules to ease a potential port to the browser. I really don't  feel strongly about that, and am willing to remove the import if anyone disagrees.

> question: why are we prepending `'node:'` to the imported module names?

Every time we try to introduce a new module in core comes the question of how breaking it would be for the ecosystem (is there a module that use the same name out there). It's been discussed several time to add a prefix to core modules, and with ESM we had to come up with a URL scheme to be spec compliant. TSC has decided that it will be ported to the CJS resolver as well (see https://github.com/nodejs/node/issues/36098), so why not use it.
Again, I don't feel strongly about that either, if the consensus is not to use the prefix, that'd be fine with me.